0.10/Dockerfile.rhel7

```diff
@@ -32,6 +35,8 @@ COPY ./s2i/bin/ $STI_SCRIPTS_PATH
 # run and build the applications.
 COPY ./contrib/ /opt/app-root
 
+RUN $PROMPT_FILE && npm install -g nodemon
```

Comment:
this is going to be a problem for our rhel builds since everything they install has to come from supported rpms, so we'd need nodemon in a rhel repository package.